models/lpjguess/R/split_inputs.LPJGUESS.R

```diff
@@ -0,0 +1,131 @@
+## split LPJ-GUESS ncdf files into smaller time units to use in KF
```

Comment:
FYI, this bit isn't needed unless the model, like sipnet, gets it start/end date info from its met files